### ISSUE TITLE
pickPosition doesn't pick ellipsoid in 2D and CV

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#5076]
In 3D, `pickPosition` will return a position when hovering over the globe.  In 2D and CV, it's the result is always `undefined`

``` javascript 
var viewer = new Cesium.Viewer('cesiumContainer', {
    sceneMode: Cesium.SceneMode.SCENE2D
});

// Mouse over the globe to see the cartographic position
var handler = new Cesium.ScreenSpaceEventHandler(viewer.scene.canvas);
handler.setInputAction(function(movement) {
    var cartesian = viewer.scene.pickPosition(movement.endPosition);
    if (!Cesium.defined(cartesian)) {
      console.log('undefined');
    }
}, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
```

**Comment by thw0rted** [comment]
I came to report that 2D picking isn't working for WMS (GetFeatureInfo), and I'm wondering if it's a subset of this issue.  Demo: go to [the WMS Sandcastle](http://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Web%20Map%20Service%20(WMS).html&label=Showcases), select a point, watch the info load. 
 Now, switch to 2D, and click again.  You can't select anything.  This is because [this call to Globe.pick](https://github.com/AnalyticalGraphicsInc/cesium/blob/1.35/Source/Scene/ImageryLayerCollection.js#L367) is returning `undefined`.

(If this is a different issue, I can open a new ticket.)

**Comment by hpinkos** [comment]
Hey @thw0rted, that bug you're seeing is related to this issue:#2480 
I've made a comment that you ran into this problem too

**Comment by thw0rted** [comment]
Thanks, @hpinkos !  I didn't see any workarounds mentioned on the other ticket -- do I just need to tell customers that WMS picking won't work in 2D for now?

**Comment by hpinkos** [comment]
Yeah, unfortunately I can't think of a good workaround.  It's labeled high priority, but I'm not sure when we'll get a chance to look at it.  If you'd like to take a stab at a solution, comment in that issue and someone from the team can give you some tips on where to get started.

**Comment by thw0rted** [comment]
Unfortunately the "hard math" bits are my biggest weakness -- I can usually track what everything is doing but once the vectors start normalizing I have to "step over" in the debugger :D

**Comment by gadew64** [comment]
Any luck with this?  Has it been solved after 1.46?

**Comment by OmarShehata** [comment]
Reported again here: https://groups.google.com/d/msg/cesium-dev/k1brfPffEec/GJUOPXywAAAJ

**Comment by jonathanvhive** [comment]
A very useful workaround for anyone getting here:

```
var viewer = new Cesium.Viewer("cesiumContainer");

var sseh = new Cesium.ScreenSpaceEventHandler(viewer.scene.canvas);

sseh.setInputAction(function(event) {
  console.log('x', event.position);
  
  var cartesian = viewer.camera.pickEllipsoid(
    event.position,
    viewer.scene.globe.ellipsoid
  );
  
  //const pp = viewer.scene.pickPosition(event.position);
  
  if (cartesian) {
    const cart = Cesium.Cartographic.fromCartesian(cartesian);
    console.log(cart.longitude * (180 / Cesium.Math.PI));
    console.log(cart.latitude  * (180 / Cesium.Math.PI));
  }
}, Cesium.ScreenSpaceEventType.LEFT_DOWN);
```

**Comment by ggetz** [comment]
While testing after the changes in https://github.com/CesiumGS/cesium/pull/13098, I _was_ able to get results using the original code snippet. However, I could still reproduce around the antemeridian (IDL) and the poles.

The remaining issues should be covered by https://github.com/CesiumGS/cesium/issues/13180 and https://github.com/CesiumGS/cesium/issues/13179 respectively, so I am closing.
